WordPress/Classes/Utility/WebKitViewController.swift

```diff
@@ -354,4 +354,14 @@ extension WebKitViewController: WKUIDelegate {
     func webViewDidClose(_ webView: WKWebView) {
         dismiss(animated: true, completion: nil)
     }
+
+    func webView(_ webView: WKWebView, didFailProvisionalNavigation navigation: WKNavigation!, withError error: Error) {
+        DDLogInfo("\(NSStringFromClass(type(of: self))) Error Loading [\(error)]")
+
+        if !ReachabilityUtils.isInternetReachable() {
```

Comment:
I noticed that when trying this on airplane mode, the message on `error` would be the exact same one provided by `ReachabilityUtils.showAlertNoInternetConnection()`, so we might not need to handle it as a special case

Reply:
That is true (except for the alert title). However, I did this so the no connection alert would be consistent. Since we have a specific `No Connection` alert, if it changes in the future this one will be in sync.